assets/express/src/api/v1/todos/dto/replace-todo.dtm.ts

```diff
@@ -0,0 +1,3 @@
+import { putTodoInput } from 'src/modules';
```

Comment:
It seems that the extension is not right.

```suggestion
replace-todo.dto.ts instead replace-todo.dtm.ts
```